### PR TITLE
KATI-72_Appointment-Request-Detail-UI

### PR DESCRIPTION
**Pull request by vtnlinh95**

### CODE REVIEW

File: app/src/main/java/com/kms/cura/model/Settings.java [hunk 1]
```diff
@@ -4,7 +4,8 @@
  * Created by linhtnvo on 6/6/2016.
  */
 public class Settings {
-    public static final String SERVER_URL = "http://192.168.74.141:8080/cura/cura_server";
+    //public static final String SERVER_URL = "http://192.168.74.141:8080/cura/cura_server";
+    public static final String SERVER_URL = "http://10.0.3.2:8080/com.kms.cura_server/cura_server";
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/model/Settings.java]: Local URL :-1:

File: app/src/main/java/com/kms/cura/view/activity/AppointmentRequestDeatailActivity.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    private TextView loadText(int id, String text){
+        TextView textView = (TextView) findViewById(id);
+        if(text == null){
+            textView.setVisibility(View.GONE);
+        }
+        textView.setText(text);
+        return textView;
+    }
+
+    private int getLineCount(){
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/activity/AppointmentRequestDeatailActivity.java]: Re-format this file

File: app/src/main/java/com/kms/cura/view/activity/AppointmentRequestDeatailActivity.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+    @Override
+    public void onClick(View v) {
+        int id = v.getId();
+        if(id == R.id.btnSeeMore){
+            InformationDialogFragment dialogFragment = new InformationDialogFragment();
+            Bundle bundle1 = new Bundle();
+            bundle1.putString(AppointmentRequestDeatailActivity.COMMENT,bundle.getString(DoctorRequestListFragment.COMMENT));
+            dialogFragment.setArguments(bundle1);
+            dialogFragment.show(getFragmentManager(),"dialog");
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/activity/AppointmentRequestDeatailActivity.java]: "dialog" should be made constant

File: app/src/main/java/com/kms/cura/view/fragment/DoctorRequestListFragment.java [hunk 4]
```diff
@@ -29,11 +38,20 @@
...
+    public static String APPT_TIME = "APPT_TIME";
+    public static String APPT_FACILITY_NAME = "APPT_FACILITY_NAME";
+    public static String COMMENT = "COMMENT";
+    public static String APPT_FACILITY_ID = "APPT_FACILITY_ID";
+
     private ExpandableStickyListHeadersListView lvRequestList;
     private SwipeRefreshLayout mSwipeRefreshLayout;
     private DoctorRequestListAdapter adapter;
     private WeakHashMap<View,Integer> mOriginalViewHeightPool = new WeakHashMap<View, Integer>();
+    private List<AppointmentEntity> requestList;
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/fragment/DoctorRequestListFragment.java]: Do you even need this as field ?

File: app/src/main/java/com/kms/cura/view/fragment/DoctorRequestListFragment.java [hunk 5]
```diff
@@ -96,4 +116,35 @@ private void modifyToolbar() {
...
+        Intent toDetail = new Intent(getActivity(), AppointmentRequestDeatailActivity.class);
+        toDetail.putExtras(createBundle(appt));
+        startActivity(toDetail);
+    }
+
+    private Bundle createBundle(AppointmentEntity appt) {
+        Bundle bundle = new Bundle();
+        PatientUserEntity patientUserEntity = appt.getPatientUserEntity();
+        FacilityEntity facilityEntity = appt.getFacilityEntity();
+        bundle.putString(PATIENT_ID,patientUserEntity.getId());
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/fragment/DoctorRequestListFragment.java]: re-format this file

File: app/src/main/java/com/kms/cura/view/fragment/InformationDialogFragment.java [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+        AlertDialog.Builder builder = createDialogBuilder(root);
+        txtComment = (TextView) root.findViewById(R.id.txtComment);
+        txtComment.setText(text);
+        return builder.create();
+    }
+
+    private AlertDialog.Builder createDialogBuilder(View root) {
+        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
+        builder.setView(root);
+        builder.setNeutralButton("OK", new DialogInterface.OnClickListener() {
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/fragment/InformationDialogFragment.java]: Put "OK" in resource file

**Comment by hieumai** [comment]
OK from me. @huuduy216  and @toannguyen71195  please start verifying on this

**Comment by huuduy216** [comment]
@vtnlinh95  grammar error when there's conflict ( "This conflictS")

**Comment by hieumai** [comment]
@vtnlinh95 please resolve conflicts and rebase so I can merge

**Comment by vtnlinh95** [comment]
@hieumai rebased

**Pull request merged**